lib/Distribution/Opensuse/Leap/16Latest.pm

```diff
@@ -14,10 +14,25 @@ use strict;
 use warnings FATAL => 'all';
 
 use Yam::Agama::Pom::GrubMenuLeapPage;
+use Yam::Agama::Pom::GrubMenuAgamaPage;
+use Yam::Agama::Pom::GrubMenuAgamaDeprecatedEntryOrderPage;
+use testapi qw(record_soft_failure);
 
 sub get_grub_menu_installed_system {
     my $self = shift;
     return Yam::Agama::Pom::GrubMenuLeapPage->new({grub_menu_base => $self->get_grub_menu_base()});
 }
 
+sub get_grub_menu_agama {
+    if (is_ppc64le()) {
```

Comment:
This seems to cause failures in Leap: https://openqa.opensuse.org/tests/5260957#

Reply:
I filed PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/23082 and https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/23083 to fix this issue.